src/app/containers/CpsRecommendations/RecommendationsPromoImage/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shape, bool } from 'prop-types';
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+import ImageWithPlaceholder from '../../ImageWithPlaceholder';
+import { createSrcset } from '#lib/utilities/srcSet';
+import getOriginCode from '#lib/utilities/imageSrcHelpers/originCode';
+import getLocator from '#lib/utilities/imageSrcHelpers/locator';
+
+const RecommendationsImage = ({ indexImage, lazyLoad }) => {
+  if (!indexImage) {
+    const landscapeRatio = (9 / 16) * 100;
+    return <ImagePlaceholder ratio={landscapeRatio} />;
+  }
+
+  const { height, width, path } = indexImage;
+
+  const ratio = (height / width) * 100;
+  const originCode = getOriginCode(path);
+  const locator = getLocator(path);
+  const imageResolutions = [70, 95, 144, 183, 240, 320, 660];
+  const srcset = createSrcset(originCode, locator, width, imageResolutions);
+  const sizes = '(max-width: 1008px) 33vw, 321px';
```

Comment:
I think this should be rem too - 1008px is 63rem, which is GEL_GROUP_4_SCREEN_WIDTH_MIN